apps/src/code-studio/pd/workshop_dashboard/EnrollmentsPanel.jsx

```diff
@@ -90,18 +98,50 @@ export default class EnrollmentsPanel extends React.Component {
     })
       .done(() => {
         // reload
-        loadEnrollments();
+        this.handleEnrollmentRefresh();
         this.moveEnrollmentRequest = null;
       })
       .fail(() => {
         this.setState({
           error: 'Error: unable to move enrollments'
         });
-        loadEnrollments();
+        this.handleEnrollmentRefresh();
         this.moveEnrollmentRequest = null;
       });
   };
 
+  handleEditEnrollmentConfirmed = updatedName => {
+    this.handleEditEnrollment(updatedName, this.state.selectedEnrollments[0]);
+    this.setState({
+      enrollmentChangeDialogOpen: null,
+      selectedEnrollments: []
+    });
+  };
+
+  handleEditEnrollment = (updatedName, selectedEnrollment) => {
+    let updatedNameSnakeCase = {
+      first_name: updatedName.firstName,
+      last_name: updatedName.lastName
+    };
+
+    this.editEnrollmentRequest = $.ajax({
+      method: 'POST',
+      url: `/api/v1/pd/enrollment/${selectedEnrollment.id}/edit`,
+      contentType: 'application/json',
+      data: JSON.stringify(updatedNameSnakeCase)
```

Comment:
does it not work to send the data as an object? Ex:
```
data: {
      first_name: updatedName.firstName,
      last_name: updatedName.lastName
}
```
Then you wouldn't need the extra variable and JSON call

Reply:
It seems like you have to[ serialize your JS object into a string](https://stackoverflow.com/questions/12693947/how-to-send-json-instead-of-a-query-string-with-ajax) (...did some reading about difference between JSON and a JS object this morning) to send via ajax...I could just do this all in one line I suppose instead of creating a variable...